domain/schema/controller/sql/0022-model-authorized-keys.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE model_authorized_keys (
+    model_id TEXT NOT NULL,
+    user_public_ssh_key_id INTEGER NOT NULL,
```

Comment:
instead of the index below, why not define a composite primary key like we do elsewhere?

Reply:
Composite primary key's don't work with our trigger generation logic.

Reply:
Had this discussion today and demonstrated the problem with composite primary keys. Will talk with Simon more about it.

Reply:
All fixed.